test/common.js

```diff
@@ -4,40 +4,40 @@
  */
 
 var referee = require('referee')
-  , assert  = referee.assert
-  , refute  = referee.refute
-  , crypto  = require('crypto')
-  , async   = require('async')
-  , rimraf  = require('rimraf')
-  , fs      = require('fs')
-  , path    = require('path')
-  , delayed = require('delayed').delayed
-  , levelup = require('../lib/levelup.js')
-  , errors  = require('level-errors')
-  , dbidx   = 0
-
-assert(levelup.errors === errors);
+var assert = referee.assert
+var refute = referee.refute
+var crypto = require('crypto')
+var async = require('async')
+var rimraf = require('rimraf')
+var fs = require('fs')
+var path = require('path')
+var delayed = require('delayed').delayed
+var levelup = require('../lib/levelup.js')
+var errors = require('level-errors')
+var dbidx = 0
+
+assert(levelup.errors === errors)
 
 referee.add('isInstanceOf', {
-    assert: function (actual, expected) {
-        return actual instanceof expected
-    }
-  , refute: function (actual, expected) {
-        return !(actual instanceof expected)
-    }
-  , assertMessage: '${0} expected to be instance of ${1}'
-  , refuteMessage: '${0} expected not to be instance of ${1}'
+  assert: function (actual, expected) {
+    return actual instanceof expected
+  },
+  refute: function (actual, expected) {
+    return !(actual instanceof expected)
+  },
+  assertMessage: '${0} expected to be instance of ${1}', // eslint-disable-line
+  refuteMessage: '${0} expected not to be instance of ${1}' // eslint-disable-line
```

Comment:
Had to add this exception since standard thinks we are trying to use a es6 template string, which we aren't.